Install error `go get`: need to use `go install` from version 1.8

バグというより、仕様変更。
Go 1.8 より、go get ではなくて、go install を使って、go アプリはインストールしてねに変わったため、error になった。
ちなみに、go get は、go.mod の依存関係を調整することに特化するようになった。

```
a@as-macbook-air dotfiles % make install ROLE=vim
2022-03-21 15:15:30 [INFO] Install vim...
2022-03-21 15:15:30 [INFO] Install > homebrew...
2022-03-21 15:15:30 [INFO] Install > fzf...
2022-03-21 15:15:31 [INFO] Install > ripgrep...
2022-03-21 15:15:32 [INFO] Install > memo...
go: go.mod file not found in current directory or any parent directory.
	'go get' is no longer supported outside a module.
	To build and install a command, use 'go install' with a version,
	like 'go install example.com/cmd@latest'
	For more information, see https://golang.org/doc/go-get-install-deprecation
	or run 'go help get' or 'go help install'.
make: *** [install] Error 1
a@as-macbook-air dotfiles % go version
go version go1.18 darwin/amd64
```

version も指定する必要がある。

```
2022-03-21 16:29:02 [INFO] Install > memo...
go: 'go install' requires a version when current directory is not in a module
	Try 'go install github.com/mattn/memo@latest' to install the latest version
make: *** [install] Error 1
```

--- a/roles/memo/install.sh
+++ b/roles/memo/install.sh
@@ -5,7 +5,7 @@ readonly CURRENT_PATH=$(cd $(dirname $0); pwd)
 
 
 type memo > /dev/null 2>&1 || {
-  go get github.com/mattn/memo
+  go install github.com/mattn/memo@latest
 }
 
 ( 

